pkg/skaffold/docker/reference_test.go

```diff
@@ -85,15 +85,15 @@ func TestParseReference(t *testing.T) {
 		{
 			description: "invalid reference",
 			image:       "!!invalid!!",
-			shoudErr:    true,
+			shouldErr:   true,
```

Comment:
I use `gofmt`, I hope that's the right thing tm.

Disclaimer: I'm using `hg` w/ `hg fix` to maintain this work, otherwise dealing w/ such stuff is really painful. That means that I may temporarily mess up the PR when I go to try to fix things up by dropping intermediate commits. I will try very hard not to, but ... please bear with me.